### PR TITLE
/samples/upsert/bulk should return the job id too

### PR DESCRIPTION
**Pull request by shriramshankar**

### CODE REVIEW

File: api/v1/swagger.yaml [hunk 1]
```diff
@@ -6854,7 +6854,14 @@ definitions:
...
           Status.
-
+      jobId:
+        type: integer
+        readOnly: true
+        description: >
+          Optionally a jobId is returned with the response when the bulk upsert
+          request is enqueued as a job for the workers to process it
+          asynchronously. The returned jobId can be used to check the status
+          of the status of the bulk upsert request.
```
Comment (iamigo) [inline api/v1/swagger.yaml]: "the status of the status of"

**Comment by shriramshankar** [comment]
@iamigo @annyhe Updated the PR.

**Comment by shriramshankar** [comment]
@annyhe @iamigo Can we merge this if no further changes are needed? I am close to creating a PR for the 'invalid range limit error'.

**Comment by shriramshankar** [comment]
@annyhe @iamigo this PR also fixes the "Invalid string length error".

**Pull request merged**